Xamarin.Forms.Core/Page.cs

```diff
@@ -191,6 +193,13 @@ public Task<bool> DisplayAlert(string title, string message, string accept, stri
 			return args.Result.Task;
 		}
 
+		public Task<string> DisplayPromptAsync(string title, string message, string accept = "OK", string cancel = "Cancel", string placeholder = null, int? maxLength = null, Keyboard keyboard = default(Keyboard))
```

Comment:
don't use a nullable int, but a value of 0 or -1 to indicate that you don't want any limit